vmdb/app/models/host.rb

```diff
@@ -1707,6 +1707,12 @@ def scan_from_queue(taskid=nil)
 
       # Skip SSH for ESXi hosts
       unless self.is_vmware_esxi?
+        unless self.hostname
```

Comment:
This is only checking for nil.  Would be better off with `if hostname.blank?`.  Otherwise the change looks good.